Time-Card/DRV/ptp_ocp.c

```diff
@@ -3923,6 +3926,132 @@ config_write(struct file *filp, struct kobject *kobj,
 }
 static BIN_ATTR_RW(config, OCP_CONFIG_SIZE);
 
+static ssize_t
+art_config_read(struct file *filp, struct kobject *kobj,
```

Comment:
Can this be named something like 'eeprom_config_read'?  Things in the sysfs space become an API, and this isn't really art_ specific.

Reply:
ok, I will remove flash version, but the size of the config, and the offset is really specific to the art-card. We don't have any config in the eeprom in OCP Time Card

Reply:
I ended up renaming new attributes to `disciplining_config` and `temperature_table` to reflect the actual meaning. Flash config is still in-place, will remove with another PR